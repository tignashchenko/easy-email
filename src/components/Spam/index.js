import React from 'react';

const Spam = () => (
    <div>
        <h1>This is spam!</h1>
    </div>
);

export default Spam;
