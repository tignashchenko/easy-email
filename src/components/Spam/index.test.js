import React from 'react';
import dom from 'react-test-renderer';
import Spam from './';

const renderTree = dom.create(<Spam />).toJSON();

describe('Spam component', () => {
    test('Spam component should correspond to its snapshot counterpart', () => {
        expect(renderTree).toMatchSnapshot();
    });
});
