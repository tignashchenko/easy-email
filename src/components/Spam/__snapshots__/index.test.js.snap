// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spam component Spam component should correspond to its snapshot counterpart 1`] = `
<div>
  <h1>
    This is spam!
  </h1>
</div>
`;
